### PR TITLE
More contributors

### PR DESCRIPTION
**Pull request by privat**
add metadata `package.more_contributors` for lib/core and lib/nitcorn.

### CODE REVIEW

File: lib/nitcorn/package.ini [hunk 1]
```diff
@@ -3,6 +3,7 @@ name=nitcorn
 tags=network,lib
 maintainer=Alexis Laferrière <alexis.laf@xymus.net>
 license=Apache-2.0
+more_contributors=Jean-Philippe Caissy <jean-philippe.caissy@shopify.com>, Guillaume Auger, Frederic Sevillano, Justin Michaud-Ouellette, Stephan Michaud, Maxime Bélanger
```
Comment (xymus) [inline lib/nitcorn/package.ini]: @Morriar should probably be in this list, and @R4PaSs.

Comment (privat) [inline lib/nitcorn/package.ini]: more_contributors are for additional people not present in the git history

2015-11-07 11:00 GMT-05:00 Alexis Laferrière <notifications@github.com>:

Comment (xymus) [inline lib/nitcorn/package.ini]: Ha ok.

**Comment by xymus** [comment]
+1

**Pull request merged**